README.adoc

```diff
@@ -81,30 +81,30 @@ You will update the methods in the template interface for this client to be asyn
 
 [role="code_command hotspot", subs="quotes"]
 ----
-#Replace the `JobGatewayClient` interface.#
-`gateway/src/main/java/io/openliberty/guides/gateway/client/JobGatewayClient.java`
+#Replace the `JobClient` interface.#
+`gateway/src/main/java/io/openliberty/guides/gateway/client/JobClient.java`
 ----
-JobGatewayClient.java
+JobClient.java
 [source, Java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/gateway/src/main/java/io/openliberty/guides/gateway/client/JobGatewayClient.java[]
+include::finish/gateway/src/main/java/io/openliberty/guides/gateway/client/JobClient.java[]
 ----
 
 The changes involve changing the return types of the [hotspot=getJobs file=0]`getJobs`, [hotspot=getJob file=0]`getJob`, and [hotspot=createJob file=0]`createJob` methods to return `CompletionStage<T>` types. These changes make the methods asynchronous. Since the methods now have return type of `CompletionStage<T>`, you aren't able to directly manipulate the inner types. As you will see in the next section, you will be able to indirectly use the inner object by chaining callbacks.
 
-== Responding asynchronously to an HTTP request
+== Updating REST resource to handle HTTP requests asynchronously
 
 JAX-RS resources can also have asynchronous methods. So instead of returning a `JobsList` model type, you can return a `CompletionStage<JobsList>` type. Completion stages can be chained together by using the [hotspot=thenApplyAsync file=0]`thenApplyAsync()` method.
 
 [role="code_command hotspot", subs="quotes"]
 ----
-#Replace the `JobGatewayResource` class.#
-`gateway/src/main/java/io/openliberty/guides/gateway/JobGatewayResource.java`
+#Replace the `JobResource` class.#
```

Comment:
The job resource class for the `gateway` service needs to be renamed, because it's duplicate with the `job/src/main/java/io/openliberty/guides/job/JobResource.java` for the `job` resource.

Reply:
Looks like you had renamed the class in the code to `GatewayJobResource`?

Reply:
Good catch, this has been corrected now